experimental/docker-compose/docker-compose.yml

```diff
@@ -49,6 +49,7 @@ gate:
     - SERVICES_FRONT50_HOST=front50
     - SERVICES_IGOR_HOST=igor
     - SERVICES_ORCA_HOST=orca
+    - SERVICIES_ROSCO_HOST=rosco
```

Comment:
Bah! I had copy my changes I have been running on a demo instance of spinnaker so obviously I mistyped it as when trying to created this PR :(